### DISCUSSION TITLE
G7 updates only every ~15 minutes

### DISCUSSION THREAD

**Discussion by rudiru** [NightscoutFoundation/xDrip#3222] [Bug reports]
Hi Team,
I'm having serious issues on my BG reading updates after i left the phone out of range of the G7 for a 10 minutes.
Eversince the values are not updated regularly anymore and it takes about 15 minutes to get a new reading. The previous values are backfilled in xdrip however it is altogether unreliable. I have been using the settings from [here](https://navid200.github.io/xDrip/docs/Dexcom/G7.html)  and i am on nightly updates (5a5681d-2023.11.29)

In system status 
Voltage A is 288
Voltage B is 263

BR

Ru

**Comment by Navid200** [comment]
What is the phone brand and model?
What is the Android version?

**Reply by Navid200** [reply]
These are some of the actions that may help address intermittent connectivity.
https://navid200.github.io/xDrip/docs/Intermittent.html

**Reply by rudiru** [reply]
Thanks!
The issue has resolved itself after one day, the values now come in regularly.
I have removed old G6 transmitters from the BT devices and recoupled my Bose BT earphones.
This didnt seem to have any immidiate effect but now about 24h later the readings are updated regularly again.

Phone S23 Plus
Android 14

Voltage B has dropped to 259 by now.

issue is solved my side.
